packages/taste-test/src/lib/classes/TasteTest.class.ts

```diff
@@ -337,38 +383,64 @@ export class TasteTest implements AbstractTasteTest {
       ownNode = nodeUTXOS[0];
     } else {
       const nodeUTXOs = await this.lucid.utxosAt(nodeValidatorAddr);
-      ownNode = findOwnNode(nodeUTXOs, userKey);
+      ownNode = findOwnNode(
+        nodeUTXOs,
+        userKey,
+        this._getTasteTestTypeFromArgs(args).tasteTestType
```

Comment:
this seems a little awkward for a few reasons
1) why not just do `args.tasteTestType ?? "basic"`, or require `tasteTestType` be set?
2) this is called _getTasteTestTypeFromArgs, but it returns the args themselves, modified to include the type

Reply:
I updated it to just call a method. The reason I am using a method is cause I need a fallback in like 10 places.